mm2src/komodefi_cli/src/config.rs

```diff
@@ -81,21 +81,6 @@ impl KomodefiConfig for KomodefiConfigImpl {
     fn orderbook_volume_precision(&self) -> &SmartFractPrecision { &VOLUME_PRECISION }
 }
 
-impl Display for KomodefiConfigImpl {
-    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        if !self.is_set() {
-            return writeln!(f, "komodefi configuration is not set");
-        }
-        writeln!(
-            f,
-            "mm2 RPC URL: {}",
-            self.rpc_uri.as_ref().expect("Expected rpc_uri is set")
-        )?;
-        writeln!(f, "mm2 RPC password: *************")?;
-        Ok(())
-    }
-}
```

Comment:
You can keep the `Display` implementation instead of replacing it with `print_config`.

One example would be adding `print_password_on_tty` in `KomodefiConfigImpl`, create a function like `mask_password_on_tty(&self) -> Self` and use that on printing the value if needed.

Reply:
done.  impl `print_password_on_tty`

Reply:
By the way, you can remove the `on_tty` part from the function name. I was suggesting it as a struct field where `on_tty` makes it more clear. For function, you don't need that part (it can be even confusing IMO).

Reply:
Build fails, you forgot to update function in config.rs